README.md

```diff
@@ -13,8 +13,8 @@ We adhere to the [Ember Community Guidelines](https://emberjs.com/guidelines/) f
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.8 or above
-* Ember CLI v2.13 or above
+* Ember.js v3.20 or above
+* Ember CLI v3.20 or above
```

Comment:
Just curious, what change caused this compatibility requirement to be upgraded?

Reply:
https://github.com/ember-cli/ember-cli/pull/9491